Resources/Prototypes/DeltaV/Entities/Clothing/Hands/gloves.yml

```
@@ -41,3 +41,23 @@
     damage:
       types:
         Blunt: 9.5
+
+- type: entity  #Delta-V heavy duty chemistry gloves
```

```suggestion
- type: entity
```
No need for a change marker since it is in a DeltaV file.